README.md

```diff
@@ -457,6 +494,43 @@ class DeliverViaSMSC
   end
 end
 ```
+
+### Custom Retry Queues
+
+By default, when a job is retried, it is added to the `@queue` specified in the worker. However, you may want to push the job into another (lower or higher priority) queue when the job fails. You can do this by dynamically specifying the retry queue. For example:
+
+```ruby
+class ExampleJob
+  extend Resque::Plugins::Retry
+  @queue = :testing
+  @retry_delay = 1
+
+  def self.work(user_id, user_mode, record_id)
+    Resque.enqueue_to(
+      target_queue_for_args(user_id, user_mode, record_id),
+      self,
+      user_id, user_mode, record_id
+     )
+  end
+
+  def self.retry_queue(exception, *args)
+    target_queue_for_args(*args)
+  end
+
+  def self.perform(*args)
+    raise
```

Comment:
For the sake of the example, I don't think this _should_ be a `raise` instead do something similar to `heavy_lifting` above

Reply:
Agreed.